src/pretix/presale/templates/pretixpresale/event/index.html

```diff
@@ -442,7 +442,7 @@ <h3>{{ tup.0.name }}</h3>
             </form>
         {% endif %}
     {% endif %}
-    {% if vouchers_exist %}
+    {% if vouchers_exist and not event.has_subevents or vouchers_exist and subevent %}
```

Comment:
This is complicated logic, so I think I'm going to rename `vouchers_exist` to `show_vouchers` and add the logic to the python.